### ISSUE TITLE
Rename module to exclude demo

### ISSUE THREAD

**Issue by Woland2k** [VirtoCommerce/vc-module-demo-customer-segments#1]
Module repository name shouldn't include "demo", the module should be released in preview mode instead (include preview in the name).

**Comment by mvktsk** [comment]
Task https://virtocommerce.atlassian.net/browse/VP-6282 has been created

**Comment by asvishnyakov** [comment]
@Woland2k This is not preview of platform module, it's module for demo solution (private & public demo). All our modules has demo prefix, it's naming convention. So "Won't fix"

**Comment by Woland2k** [comment]
@asvishnyakov the point is that we shouldn't used demo prefix for repo/module names. The modules that are not production ready should be marked differently, for instance with a "preview" suffix in the name (not repo name).

**Comment by pushnitsa** [comment]
@Woland2k This repositories are not for production using. They not included in modules.json which uses for modules installation.
This repos only for demo purposes, and developing by Demo Solution Team (this means demo- prefix in repo name), and now we have 6 repos:
https://github.com/VirtoCommerce?q=demo

**Comment by Woland2k** [comment]
@pushnitsa yes I'm aware that it is not for production, ideally there should be a tag or category called "preview" for modules that are not production quality, it will still allow others to take over and possibly make it production if needed.

**Comment by asvishnyakov** [comment]
@Woland2k This is not preview, it will NEVER ready for production. Others also shouldn't take it over and use any way, this is not sample.

**Comment by Woland2k** [comment]
@asvishnyakov demo or sample are quite the same concepts, lets schedule a discussion about this

**Comment by pushnitsa** [comment]
@Woland2k sure, which time would be convenient to you?
